src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -1,30 +1,60 @@
-export const TodoFilter = () => (
-  <form className="field has-addons">
-    <p className="control">
-      <span className="select">
-        <select data-cy="statusSelect">
-          <option value="all">All</option>
-          <option value="active">Active</option>
-          <option value="completed">Completed</option>
-        </select>
-      </span>
-    </p>
+import { Dispatch, SetStateAction } from 'react';
+import { FilterType } from '../../types/FilterType';
 
-    <p className="control is-expanded has-icons-left has-icons-right">
-      <input
-        data-cy="searchInput"
-        type="text"
-        className="input"
-        placeholder="Search..."
-      />
-      <span className="icon is-left">
-        <i className="fas fa-magnifying-glass" />
-      </span>
+type Props = {
+  setFilterBy: Dispatch<SetStateAction<FilterType>>;
```

Comment:
Not critical, but I would use 

```suggestion
  setFilterBy: (newFilterType: FilterType) => void;
```

Reply:
It is because in fact you don't need everything from React.Dispatch